src/components/fileStorage/repo/files.repo.js

```diff
@@ -17,7 +17,13 @@ const personalFileSearchQuery = (userId) => ({
 	owner: userId,
 });
 
-const getFileById = async (id) => FileModel.findById(id).lean().exec();
+const getFileById = async (id) => {
+	const file = await FileModel.findById(id).lean().exec();
+	if (!file) throw new NotFound();
+	return file;
+};
+
+const getFileOrDeletedFileById = async (id) => FileModel.findOneWithDeleted({ _id: id });
 
```

Comment:
where we use this?

Reply:
The function was moved to the repo integration test. The place where this function is used.